content/ru/_catalog/controllers/wiren-board-8.md

```
@@ -0,0 +1,77 @@
+::product
+
+#description
+Wiren Board 8 — это универсальный модульный контроллер автоматизации, работающий под управлением свободного программного обеспечения.
+
+Контроллеры Wiren Board применяются в задачах мониторинга серверного и климатического оборудования, диспетчеризации и сбора данных с приборов учёта, в качестве основы для «умного дома» и автоматизации производств. [Примеры внедрений](https://wirenboard.com/ru/contents/solutions/).
+
+Основные отличия от [Wiren Board 7](https://wirenboard.com/ru/product/wiren-board-7/): 64-битный ARM процессор 1.5 ГГц 28 нм, до 4 Гбайт оперативной памяти, а CAN теперь модулем расширения.
+**В ревизии 8.4 в MOD1 можно установить только модули дискретных входов и выходов, MOD2 и MOD3 без ограничений, а MOD4 недоступен.**
+
+#info
+![](img/catalog/controllers/wiren-board-8/wide.png)
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)